web/css/main.css

```diff
@@ -155,6 +160,54 @@ a,
     line-height: 1.35;
 }
 
+/* Services table */
+tr.service:hover {
+    background-color: initial !important;
+}
+.service-name {
+    width: 100%;
+    color: #4db3cf;
+    display: block;
+}
+.service-details-state-toggle {
+    cursor: pointer;
```

Comment:
How perceptive of you to have placed it here, rather than on `.service-name` :100: